lib/ecto/changeset.ex

```diff
@@ -287,6 +287,8 @@ defmodule Ecto.Changeset do
   The function is meant for working with data internal to the application.
   Because of that neither validation nor casting is performed. This means
   `change/2` expects the keys in the `changes` map or keyword to be atoms.
+  If any value in the given `changes` does not match the
+  type for that key in the schema (or `types` map), an error is raised.
```

Comment:
I repeated this section here (`change/2`) and `put_change` and `update_change` - is this OK? Also, I kind of wanted to mentioned that an arbitrary key can be passed (that's not in schema or types) and it won't be type checked and ignored when persisting. But not sure how to best write it.